### PR TITLE
add rel tags to profile links to discourage spam

### PR DESCRIPTION
**Pull request by elbotho**
Sorry, I thought we have this for profiles already, but it was only on comments. 
This should in theory discourage spammers since there is no more SEO advantage to be gained. (But since they also use the chat now… idk 😩)

@kulla or @Entkenntnis could one of you review this? 
Unfortunately we can't really test it in staging since the descriptions are missing.

### CODE REVIEW

**Comment by Entkenntnis** [comment]
Thank you, looks good for me.

**Review by Entkenntnis — APPROVED** [review]

File: src/schema/article-renderer.tsx [hunk 1]
```diff
@@ -171,8 +171,10 @@ function renderElement(props: RenderElementProps): React.ReactNode {
   const { element, children, path } = props
 
   if (element.type === 'a') {
+    const isOnProfile =
+      path && typeof path[0] === 'string' && path[0].startsWith('profile')
```
Comment (kulla) [inline src/schema/article-renderer.tsx:175]: Out of curiosity: Where is path set?

**Review by kulla — APPROVED** [review]
LGTM and works :-)

**Comment by kulla** [comment]
> Unfortunately we can't really test it in staging since the descriptions are missing.

I tested it locally, it works

File: src/schema/article-renderer.tsx (hunk 1 above)
Comment (elbotho) [reply src/schema/article-renderer.tsx:175]: https://github.com/serlo/frontend/blob/staging/src/components/pages/user/profile.tsx#L90
It's a parameter of `renderArticle`

**Comment by elbotho** [comment]
> I tested it locally, it works

How? :)

File: src/schema/article-renderer.tsx (hunk 1 above)
Comment (elbotho) [reply src/schema/article-renderer.tsx:175]: quite handy feature, used it a lot in #1133

**Comment by elbotho** [comment]
Thank you!

**Pull request merged**